packages/@webex/internal-plugin-mercury/test/unit/spec/socket.js

```diff
@@ -530,13 +540,13 @@ describe('plugin-mercury', () => {
 
       it('sends JSON.stringifyable object', () => {
         socket.send({
-          json: true
+          json: true,
         });
         assert.calledWith(mockWebSocket.send, '{"json":true}');
       });
     });
 
-    describe('#onclose()', () => {
+    /* describe('#onclose()', () => {
```

Comment:
same again, I'm not going to add any more comments about skipped tests, because there are just too many of them, all of them need to be changed to `.skip()`